Makefile

```diff
@@ -72,8 +72,12 @@ qemu-esp: ovmf_fw.fd ovmf_vars.fd
 qemu-full: ovmf_fw.fd ovmf_vars.fd
 	# NOTE: swtpm normally exits when qemu exits, no need to cleanup
 	[ -d .tpm2 ] || mkdir .tpm2
+	rm -f .tpm2/swtpm-sock
+	killall swtpm
```

Comment:
i'd like that redirected to /dev/null so we don't have "no swtpm around" errors.... and won't `killall swtpm` exit non-zero (and fail make) if there is no 'swtpm' ?  And *then* we don't really want to kill *all* swtpm, just this one.

```shell
$ killall swtpm; echo $?
swtpm: no process found
1
```

Reply:
> won't killall swtpm exit non-zero (and fail make) if there is no 'swtpm'

Maybe.  I must have added that while pushing the patch because the tree I use does not have that killall.

(This is why I don't like making tons of stylistic changes as part of review. The final version won't be nearly as well tested).

So I'll drop that, thanks (even though || true would of course fix that specific problem)